script/Ciel/Day 14/14_01B_03_CIEL14B_4_1D.txt

```diff
@@ -312,6 +312,7 @@ I'll be borrowing your left hand."
 -- なに、<引力|アトラクション>だよ、引力。
 // Both of these 引力s are voiced as アトラクション
 // Need to use second fontbank for italics instead of PUA here for ruby
+// The quotation mark before "gravity" should be an opening quote, not a closing quote
```

Comment:
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/36a216fe-08b0-48a4-907e-62550f1b880d)
was this ever fixed?

Reply:
Yeah this was the font issue where ja2 never got updated, I think it should be fixed now

Reply:
Yeah this was already fixed. Still looks lame because of the font, but it's less lame.
![2023-09-04-201920_1500x253_scrot](https://github.com/Tsukihimates/Tsukihime-Translation/assets/2476799/c58a7546-c8f4-46fb-999d-b571c166e71d)